k8s/edge/start.sh

```diff
@@ -17,4 +17,5 @@
 
 /usr/local/openresty/bin/openresty
 
-consul-template -template="/nginx.conf.ctmpl:/usr/local/openresty/nginx/conf/nginx.conf:/usr/local/openresty/bin/openresty -s reload"
+legion-template "/nginx.conf.ltmpl" "/usr/local/openresty/nginx/conf/nginx.conf" --signal=1 --pid-file="/usr/local/openresty/nginx/logs/nginx.pid" --command="echo Reloading Nginx"
```

Comment:
SIGHUP

Reply:
Yeah, sighup is equivalent to 1 (http://meinit.nl/the-3-most-important-kill-signals-on-the-linux-unix-command-line)

Reply:
We should add str-to-int mapping (with str/choices argument)?